pages/common/rip.md

```diff
@@ -0,0 +1,28 @@
+# rip
+
+> Remove files or directories by sending them to the graveyard, allowing for them to be recovered.
+> More information: <https://github.com/nivekuil/rip>.
+
+- Remove files or directories from arbitrary locations and place them in the graveyard:
+
+`rip {{path/to/file_or_directory}} {{path/to/another/file_or_directory}}`
+
+- Interactively remove files or directories, with a prompt before every removal:
+
+`rip -i {{path/to/file_or_directory}} {{path/to/another/file_or_directory}}`
+
+- List all files in the graveyard that were somewhere in the current directory:
+
+`rip -s`
+
+- Permanently delete every file and directory in the graveyard:
+
+`rip -d`
+
+- Undo the last removal:
+
+`rip -u`
+
+- Undo all removals that are listed by `rip -s`:
```

Comment:
```suggestion
- Replace all files that are listed by `rip -s`:
```

Reply:
The `-u` stands for [U]ndo, so it might be worth it to keep it as "Undo". If you still want to get rid of undo, "Put back" might be more specific than "Replace".

Reply:
In my opinion, using replace is a clearer way of describing the command. However I like the usage of `Put back` feel free to change to that.

Reply:
That sounds fair. These changes have been made.